src/util.cpp

```diff
@@ -1886,7 +1890,15 @@ QString GetVersionAndNameStr ( const bool bWithHtml )
         strVersionText += "\n *** ";
     }
 
-    strVersionText += QCoreApplication::tr ( "Released under the GNU General Public License (GPL)" );
+    strVersionText += QCoreApplication::tr ( "Released under the GNU General Public License version 2 (GPLv2)" );
```

Comment:
I don't think that's correct - the license is GPL v2 **or later**. (There are projects out there which are GPL v2 only, e.g. the Linux kernel.)

Reply:
I'm fairly sure Jamulus has _never_ said "or later" in its licence terms.

Reply:
> run from the command line in the situation we're addressing here.

Related to the bWithHtml option (if that is to be treated as only to be used for formatting), this would mean duplicating the text which is already part of the about dialog. 
And hints to a preferred solution for this?

> For Qt, according to https://doc.qt.io/qt-5/lgpl.html , I would say we can reference that page

Just to get it correct, you just mean adding the link to the line for Qt?

Reply:
I mean specify that the we use the Qt libraries, the version of those that we use in the build in question (we use two - both 5.x - I believe), and link to the Qt LGPL page given.

I also notice in `COPYING` that's not all -- can you check there to see.

(In the distant past, when _everyone_ built Jamulus from source, `COPYING` was all that was needed.  Now that we supply binaries, we need to ensure sufficient detail is provided to comply with any licence terms we specify or comply with.)

Reply:
Added more details and also changed the bWithHtml to bDisplayInGui (for clarity around the usage).
Here is a sample of the console output:
![version-text-console](https://user-images.githubusercontent.com/13550012/147729897-04fac005-341b-4210-8dc5-67ac077b377c.png)

Please note the Qt version is dynamic, so will show the version used when compiled.

Reply:
Looks great!